pkg/mountkexec/mountkexec.go

```
@@ -52,13 +52,16 @@ func MountISOPmem(pmem, mountDir string) error {
 func KexecISO(opp *webboot.Distro, dir string) error {
 	var image boot.OSImage
 	kernelPath := opp.Kernel
+
+	//kernel can be used in multiboot.Probe
+	kernel, _ := os.Open(kernelPath)
```

you need to check the error on open. Never ignore errors.